scripts/deploy/runners.bash

```diff
@@ -29,7 +29,7 @@ while true; do
       continue
     fi
     name=$(echo "$item" | jq -r '.service.name' 2>/dev/null)
-    if [[ "$name" != "$ENVIRONMENT-runner-"* ]]; then
+    if [[ ! "$name" =~ ^$ENVIRONMENT-runner-account-(default|[0-9]+)$ ]]; then
```

Comment:
old runners are following this format `production-runner-account-123`, new runners are going to be `production-runner-account-123-1`. We don't want the script to deploy new runners